conf.py

```diff
@@ -0,0 +1,52 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# http://www.sphinx-doc.org/en/master/config
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
+
+
+# -- Project information -----------------------------------------------------
+
+project = 'tensorlayer'
+copyright = '2019, lingjun liu'
+author = 'lingjun liu'
```

Comment:
@zsdonghao Could you have a check if this file is corrected written?

Reply:
What is this file for??? There exists `docs/conf.py` for the docs template.
I think this file is duplicated.